src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -157,13 +157,17 @@ class JesBackend extends Backend with LazyLogging with ProductionJesAuthenticati
   type IOInterface = JesBackend.IOInterface
 
   override def adjustInputPaths(callKey: CallKey, inputs: CallInputs, workflowDescriptor: WorkflowDescriptor): CallInputs = inputs mapValues gcsPathToLocal
-  override def adjustOutputPaths(call: Call, outputs: CallOutputs): CallOutputs = outputs mapValues gcsPathToLocal
+  override def adjustOutputPaths(call: Call, outputs: CallOutputs): CallOutputs = outputs mapValues {
+    case CallOutput(value, hash) => CallOutput(gcsPathToLocal(value), hash) }
```

Comment:
If you're going to put this over multiple lines have the closing brace on its own line